src/api/cvc4cpp.h

```diff
@@ -3003,6 +3003,18 @@ class CVC4_PUBLIC Solver
    */
   std::vector<Term> getValue(const std::vector<Term>& terms) const;
 
+  /**
+   * When using separation logic, obtain the term for the heap.
+   * @return The term for the heap
+   */
+  Term getSeparationHeapTerm() const;
```

Comment:
I think the better name for this is `getSeparationHeap`. The reason is that the heap in separation logic is a finite map (although it is expressed by a `Term`, it is not a term).

Reply:
`getSeparationHeap` sounds good to me.